mesh/ncmesh.hpp

```diff
@@ -282,6 +282,10 @@ class NCMesh
       }
    }
 
+   const Array<int>& GetVertexToKnot() const
```

Comment:
A bit pedantic, but perhaps add a comment above the function.
Also useful for doxygen I guess.